README.md

```diff
@@ -95,15 +86,14 @@ For command line help, visit our wiki page on Neat’s [command line interface](
 1. Add Neat as a dependency:
 
   ```bash
-  npm install --save bourbon-neat
+  npm install --save just-neat
```

Comment:
See? It's got a ring to it.

This is actually necessary because there is already a npm package called `neat`.